### PR TITLE
Don't link against dl on windows

### PR DESCRIPTION
**Pull request by miguelgaio**
Signed-off-by: David Wagner <david.wagner@intel.com>
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/216%23issuecomment-139527611%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/216%23issuecomment-139532632%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/216%23discussion_r39265730%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/216%23discussion_r39267894%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/216%23discussion_r39272621%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/216%23issuecomment-139558533%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/216%23issuecomment-139527611%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3A%2B1%3A%20%5Cr%5Cn%5Cr%5Cn%40krocard%20%40OznOg%20Please%20review.%22%2C%20%22created_at%22%3A%20%222015-09-11T11%3A55%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-09-11T12%3A23%3A54Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A-1%3A%20You%20removed%20the%20%60if%28UNIX%29%60%20part.%22%2C%20%22created_at%22%3A%20%222015-09-11T14%3A16%3A54Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20304a577d6eeed8b87b429ba4d9e9cf35a5511b58%20parameter/CMakeLists.txt%207%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/216%23discussion_r39265730%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22why%20did%20you%20deleted%20the%20find_library%28dl%20dl%29%20%3F%22%2C%20%22created_at%22%3A%20%222015-09-11T12%3A26%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20agree.%20%40miguelgaio%20can%20you%20make%20this%20change%20please%20%3F%22%2C%20%22created_at%22%3A%20%222015-09-11T12%3A57%3A20Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-09-11T13%3A51%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/12096004%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/miguelgaio%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20parameter/CMakeLists.txt%3AL110-120%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/216#issuecomment-139527611'>General Comment</a></b>
- <a href='https://github.com/dawagner'></a> :+1:
@krocard @OznOg Please review.
- <a href='https://github.com/krocard'></a> :+1:
- <a href='https://github.com/dawagner'></a> :-1: You removed the `if(UNIX)` part.
- [x] <a href='#crh-comment-Pull 304a577d6eeed8b87b429ba4d9e9cf35a5511b58 parameter/CMakeLists.txt 7'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/216#discussion_r39265730'>File: parameter/CMakeLists.txt:L110-120</a></b>
- <a href='https://github.com/krocard'></a> why did you deleted the find_library(dl dl) ?
- <a href='https://github.com/dawagner'></a> I agree. @miguelgaio can you make this change please ?

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/216?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/216?approve=1'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/216'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/01org/parameter-framework/pull/216'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by dawagner** [comment]
:+1: 

@krocard @OznOg Please review.

**Comment by krocard** [comment]
:+1:

File: parameter/CMakeLists.txt [hunk 1]
```diff
@@ -110,8 +110,10 @@ include_directories(
     "${PROJECT_SOURCE_DIR}/remote-processor"
     "${PROJECT_SOURCE_DIR}/parameter/log/include")
 
-find_library(dl dl)
-target_link_libraries(parameter xmlserializer remote-processor pfw_utility dl)
+if(UNIX)
+    target_link_libraries(parameter dl)
```
Comment (krocard) [inline parameter/CMakeLists.txt]: why did you deleted the find_library(dl dl) ?

Comment (dawagner) [inline parameter/CMakeLists.txt]: I agree. @miguelgaio can you make this change please ?

Comment (miguelgaio) [inline parameter/CMakeLists.txt]: :+1:

**Comment by dawagner** [comment]
:-1: You removed the `if(UNIX)` part.

**Pull request merged**